Resources/config.xml

```diff
@@ -928,5 +928,60 @@
             </options>
         </element>
 
+        <element type="button" required="false">
+            <name>separator_detailed_errors</name>
+            <label lang="en">--------------------------------------------------------------------------------------- API Errors ---------------------------------------------------------------------------------------</label>
+            <label lang="de">--------------------------------------------------------------------------------------- API-Fehler ---------------------------------------------------------------------------------------</label>
+            <label lang="es-ES">--------------------------------------------------------------------------------------- Errores de API ---------------------------------------------------------------------------------------</label>
+            <label lang="nl-NL">--------------------------------------------------------------------------------------- API-fouten ---------------------------------------------------------------------------------------</label>
+            <label lang="fr-FR">--------------------------------------------------------------------------------------- Erreurs d'API ---------------------------------------------------------------------------------------</label>
+            <label lang="it-IT">--------------------------------------------------------------------------------------- Errori API ---------------------------------------------------------------------------------------</label>
+            <options>
+                <position>30</position>
+            </options>
+        </element>
+
+        <element type="select" scope="shop" required="false">
+            <name>show_error_details</name>
+            <label lang="en">Show error details in checkout and payment</label>
+            <label lang="de">Fehlerdetails beim Checkout und bei der Zahlung anzeigen</label>
```

Comment:
might sound annoying but we should stick to short and compact feature names.
can you change it to be that "german" title (+ other languages)
and also add a tooltip description as with the other features.

also,..please add it to the existing "order" section, that it totally fine because its the scope of ordering and checkout.

you need to adjust the positions of all following configs then :)

<img width="977" alt="Shopware_5_7_13__Rev__202206221313__-_Backend__c__shopware_AG" src="https://user-images.githubusercontent.com/5579326/190130006-65b74708-b533-4cfb-8311-d2367643044e.png">